### ISSUE TITLE
Create and Design a Database

### ISSUE THREAD

**Issue by bvogl** [EnJoIsKaTe/ProjectFancy#2]
We need a Database to store and retrieve the stored data. We have to plan the Model, setup the server and design the actual Database on the server.

- [x] Create an Entity-Relationship- Model before starting to create the Database
- [x] The ER-Model needs to be reviewed and approved by all participants 
- [x] Create the Database on the Database-Server
- [x] Design the Database after the ER-Model

**Comment by bvogl** [comment]
We uploaded an ER-Model for our Database Design to our Dropbox. You can find it in ProjectBA/DB/ER_Files. Please everyone review the model and comment here on the topic.

You can open the .xml- File with draw.io

- [x] approved by @EnJoIsKaTe 
- [ ] approved by @PatrickKi 
- [ ] approved by @xpanxnex

**Comment by PatrickKi** [comment]
Field 'Content' in dt_question should be split up in question and answer, where answer can be empty

**Comment by bvogl** [comment]
I refactored the Model once again. I left content in the dt_question- Table representing the question and added another Table for the answers. This offers the possibility to store the upvotes for a certain answer. Wo probably won't need that for the Prototype but it's nice to have this already set for future expansions.

**Comment by EnJoIsKaTe** [comment]
I will close this issue now. Because It works for now thanks to @bvogl
